appinfo/application.php

```diff
@@ -55,6 +55,16 @@ public function __construct($params=[]) {
 
 			return new Controller\ViewController($c->getAppName(), $request, $userSession, $config);
 		});
+		$container->registerService('SubscriptionsProxyController', function(IAppContainer $c) {
+			$request = $c->query('Request');
+			$userSession = $c->getServer()->getUserSession();
+			$storage = $c->query('AppStorage');
+
+			return new Controller\SubscriptionsProxyController($c->getAppName(), $request, $userSession, $storage);
+		});
+		$container->registerService('AppStorage', function($c) {
+			return $c->query('ServerContainer')->getAppFolder();
```

Comment:
If I recall correctly `->getAppFolder()` is really buggy.
@BernhardPosselt Didn't you mention smth like this?